sycl/test/abi/sycl_symbols_windows.dump

```diff
@@ -766,8 +766,6 @@
 ?Clear@exception_list@_V1@sycl@@AEAAXXZ
 ?DirSep@OSUtil@detail@_V1@sycl@@2QEBDEB
 ?DisableRangeRounding@handler@_V1@sycl@@AEAA_NXZ
-?DummyModuleHandle@OSUtil@detail@_V1@sycl@@2_JB
-?ExeModuleHandle@OSUtil@detail@_V1@sycl@@2_JB

```

Comment:
Do we need to mark these changes as an ABI break? I don't imagine these are actually used externally and it's just Windows exporting more than we need, but better to be safe.

Reply:
Done, I think.